lib/storage/client-local.js

```diff
@@ -22,5 +22,8 @@ export function get(key) {
 export function set(key, value) {
     try {
         storage[`${prefixKey}${key}`] = JSON.stringify(value);
+
+        const event = new Event('storageUpdate');
```

Comment:
Why this?

Reply:
The only purpose of this is to trigger an event that will update the shopping list counter.

Reply:
OK. Since this is going to run on 3rd party websites we need to prefix these things. Could be `tjek_shopping_list_update`